src/App.vue

```diff
@@ -1,80 +1,29 @@
 <template>
-  <div id="app">
-    <v-app :dark="isDark">
-      <Navigation />
-      <v-container grid-list-md>
-        <router-view />
-      </v-container>
-      <Footer />
+    <v-app>
+        <v-content>
+            <!-- Navigation -->
+            <Navigation />
+
+            <!-- Content -->
+            <v-container grid-list-md>
+                <router-view />
+            </v-container>
+
+            <!-- Footer -->
+            <Footer />
+        </v-content>
     </v-app>
-  </div>
 </template>
 
 <script>
-import Navigation from "./components/Navigation.vue";
-import Footer from "./components/Footer.vue";
-
-import "vuetify/dist/vuetify.min.css";
+import Navigation from "@/components/layout/Navigation.vue";
+import Footer from "@/components/layout/Footer.vue";
 
 export default {
-  name: "App",
-  components: {
-    Navigation,
-    Footer
-  },
-  computed: {
-    isDark() {
```

Comment:
Don't forget to add this (later) to the rewrite

Reply:
Will add this in the next commit.